.github/workflows/reusable-ci.yaml

```diff
@@ -0,0 +1,70 @@
+name: Reusable CI
+
+on:
+  workflow_call:
+
+jobs:
+  reusable-ci:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        otp-version: [24.1.2]
```

Comment:
I think this will be specific to each project! Does this still work if it's an input?

Reply:
Oh right, that’s a reason to use inputs 🙂

Reply:
https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#onworkflow_callinputsinput_idtype

> Required if input is defined for the on.workflow_call keyword. The value of this parameter is a string specifying the data type of the input. This must be one of: **boolean**, **number**, or **string**.

😩

Reply:
I don't think there is a real use case for a matrix for our kind of applications (it's not like we are deploying a library and we want to test for multiple versions at the same time). Maybe we can simplify it a bit?

Reply:
Huh, you’re totally right 🤦‍♂️